themes/default/content/challenge/ai-architecture.md

```diff
@@ -15,7 +15,7 @@ banner_image: /images/challenge/challenge-ai-banner.png
 
 Why write your Pulumi programs yourself when you can have AI do it for you? Pulumi AI leverages large language models (LLMs) to author infrastructure as code for any architecture for any cloud in any language.
 
-For the **GPT your IaC Challenge**, your goal is to have Pulumi AI create the most complex and sophisticated piece of architecture where you can run a successful `pulumi up` with the program created by Pulumi AI. To participate in the challenge, you'll need to either visit [pulumi.com/ai](https://pulumi.com/ai) or download the [CLI](https://github.com/pulumi/pulumi-ai) so you can use Pulumi AI.
+For the **GPT Your IaC challenge**, your goal is to have Pulumi AI create the most complex and sophisticated piece of architecture where you can run a successful `pulumi up` with the program created by Pulumi AI. To participate in the challenge, you'll need to either visit [pulumi.com/ai](https://pulumi.com/ai) or download the [CLI](https://github.com/pulumi/pulumi-ai) so you can use Pulumi AI.
```

Comment:
You lowercased "challenge" here, but it's capitalized elsewhere. Should it be capitalized here, or lowercased in other places?